.github/workflows/integration-build-product-composite.yml

```diff
@@ -1267,9 +1268,11 @@ jobs:
           cp -r ../*.json ./templates/fleet/
           cp ../product-manifest.yaml ./templates/fleet/product-manifest.yaml
           export PATH="$HOME/.local/bin:$PATH"
-          integration-pipeline transform_inventory_offline_fleet --file hosts --nr_members ${{ inputs.multi_vm_fleet_number_members }}
-          sed -i "s/member1/pallet-worker1/g" ./hosts
-          sed -i "s/member2/tugger-worker2/g" ./hosts
+          
+          nb_squads=$(grep -o 'squad: [^"]*' ./templates/fleet/product-manifest.yaml | cut -d' ' -f2- | wc -w)
```

Comment:
what is this all about ^ ^ ?

Reply:
It counts the number of squads in the manifest.
@mariana-dias-alves i think you can assume the nr of squads = number of members, 1 per squad